Add cycleops and simple docs (#1796)
dfinity/portal: docs/developer-docs/setup/cycles/cycles_management_services.md

# Using a cycles management service

> Cycles management services provide canister monitoring and automated cycles topups to ensure canisters applications remain up and running and do not run out of cycles.

Reviewer: ```suggestion

## Overview 

Once deployed to the Internet Computer, developers need to pay in cycles for the compute and storage utilized by their canisters. The process of burning cycles and transferring them to a canister is referred to as "topping-up" a canister.

Cycles management services provide canister monitoring and automated cycles top-ups to ensure canisters applications remain up and running and do not run out of cycles.

Instead of manually topping up a canister via `dfx` or scripting a custom solution, cycles management services provide tested top-up automation and and canister metric insights.
```

> ## Popular Cycles Management Services

Reviewer: ```suggestion
## Popular cycles management services
```

Popular cycles management services include:

* [CycleOps](https://cycleops.dev) - On-chain, proactive, no-code canister monitoring with historical trend graphs, topup email notifications & downloadable transaction history.
* [TipJar](https://k25co-pqaaa-aaaab-aaakq-cai.ic0.app/) - Donate cycles to your favorite canisters on the Internet Computer and keep them live and healthy!

> ## Cycles Management Libraries

Reviewer: ```suggestion
## Cycles management libraries
```

### Motoko

> * [cycles-manager](https://github.com/CycleOperators/cycles-manager) - A library for managing cycles across hundreds of canisters (Sponsored by CycleOps)

Reviewer: ```suggestion
* [cycles-manager](https://github.com/CycleOperators/cycles-manager) - A library for managing cycles across hundreds of canisters (sponsored by CycleOps).
```

Author: Adjusted to say

```
* [cycles-manager](https://github.com/CycleOperators/cycles-manager) - Provides a simplified, permissioned cycles management framework for multi-canister applications (sponsored by CycleOps).
```

> ## What is a Cycles Management Service?

Reviewer: ```suggestion

```

> Once deployed to the Internet Computer, developers need to pay in cycles for the compute and storage utilized by their canisters. The process of burning cycles and transferring them to a canister is referred to as "topping-up" a canister.

Reviewer: ```suggestion

```

> Instead of manually topping up a canister via dfx or scripting a custom solution, cycles management services provide tested topup automation and and canister metric insights so you can focus on what you do best, building a hit application!

Reviewer: ```suggestion

```
